src/context/viewer-context.tsx

```diff
@@ -35,6 +35,7 @@ function useAuthedViewer() {
   const viewerId = get(viewer, 'id', null)
   const [loading, setLoading] = React.useState(true)
   const previousViewer = React.useRef(viewer)
+  const authToken = getAccessTokenFromCookie()
```

Comment:
moved this out of the `useEffect` because if the token changes we definitely want to re-run the effect, eh?